index.php

```diff
@@ -467,21 +467,23 @@
 
     <span id="translate-download" style="display:none;" hidden>Download elementary OS</span>
     <span id="translate-purchase" style="display:none;" hidden>Purchase elementary OS</span>
-    <div id="download-modal" class="dialog modal">
-      <img alt="Download elementary OS icon" src="images/icons/apps/48/system-os-installer.svg">
+
+    <dialog id="download-modal" class="dialog modal" aria-labelledby="download-modal-title">
+      <!-- <img alt="Download elementary OS icon" src="images/icons/apps/48/system-os-installer.svg"> -->
+      <img src="https://elementary.io/images/icons/apps/48/system-os-installer.svg" alt="">
```

Comment:
Images were not loading locally so I hardcoded this but needs to be changed back.